src/main/java/com/fasterxml/jackson/databind/introspect/POJOPropertiesCollector.java

```diff
@@ -1450,6 +1450,50 @@ private PropertyNamingStrategy _findNamingStrategy()
                 return pns;
             }
         }
+
```

Comment:
`HandlerInstantiator` is given through configuration and it can return any instance of `PropertyNamingStrategy`. I think we can't rescure if `HandlerInstantiator` is used.